fda-model/compute/ImpactAreaScenarioSimulation.cs

```diff
@@ -93,7 +93,7 @@ public ImpactAreaScenarioResults Compute(interfaces.IProvideRandomNumbers random
             }
             SetStageForNonExceedanceProbability();
             ComputeIterations(convergenceCriteria, randomProvider, masterseed, computeWithDamage, giveMeADamageFrequency);
-            _impactAreaScenarioResults.ParalellTestForConvergence(.95, .05);
+            _impactAreaScenarioResults.ParallelResultsAreConverged(.95, .05);
```

Comment:
I'm confused by this rename, but also the original name. Can we add a description on this method to describe better where it's output goes? It looks like it ends up updating a property on each histogram stating whether that histogram has converged or not. I got lost trying to figure out who was consuming that information. 

Basically I expected this method to return a bool based on the name, and was surprised to find a void.

Reply:
I've struggled with this method. Really good find. It seems like this method doesn't do anything:

https://github.com/HydrologicEngineeringCenter/fda-model/blob/0823e9a695c30707c4390de9690e6cd8f35dea65/fda-model/metrics/SystemPerformanceResults.cs#L222

I can't be sure because the lambda stuff is still over my head. But it looks like we get a bool returned on line 227 and nothing is done with said bool

Reply:
I've created a ticket to address this separately.